kernel/packages/scene-system/stateful-scene/SceneStateActor.ts

```diff
@@ -0,0 +1,33 @@
+import { IEngineAPI } from 'shared/apis/EngineAPI'
+import { SceneStateDefinition } from './SceneStateDefinition'
+import { Component, ComponentData, ComponentId, EntityId, StateContainer, StatefulActor } from './types'
+
+export class SceneStateActor extends StatefulActor {
```

Comment:
At the beginning of the project, it has a purpose but since this has evolved, now it doesn't have a clear purpose, I just let it as it can be helpful in the future, however, if you think we should eliminate it, I can eliminate it without problems!